torchvision/prototype/datasets/utils/_internal.py

```diff
@@ -70,14 +66,6 @@ def read_mat(buffer: io.IOBase, **kwargs: Any) -> Any:
     return sio.loadmat(buffer, **kwargs)
 
 
-def image_buffer_from_array(array: np.ndarray, *, format: str = "png") -> io.BytesIO:
```

Comment:
This is no longer needed as it always was a crutch to enable custom decoders for datasets that didn't contain encoded images in the first place.